src/Components/SceneRefreshButton/SceneRefreshButton.types.ts

```diff
@@ -0,0 +1,43 @@
+import {
+    ICalloutContentStyles,
+    IStyle,
+    IStyleFunctionOrObject,
+    ITheme
+} from '@fluentui/react';
+import { IHeaderControlButtonStyles } from '../HeaderControlButton/HeaderControlButton.types';
+import { IHeaderControlGroupStyles } from '../HeaderControlGroup/HeaderControlGroup.types';
+
+export interface ISceneRefreshButtonProps {
+    isRefreshing: boolean;
+    lastRefreshTimeInMs: number;
```

Comment:
nit: it is bothering me seeing millisecond with Ms abbreviation :p maybe using 'ms' or 'msec' by ignoring the camel casing?

Reply:
yeah, it is fine then